test/no_std/WORKSPACE.bazel

```diff
@@ -0,0 +1,54 @@
+workspace(name = "test_no_std")
+
+local_repository(
+    name = "rules_rust",
+    path = "../../",
+)
+
+local_repository(
+    name = "test_cc_common_link",
+    path = "../cc_common_link/",
+)
+
+load("@rules_rust//rust:repositories.bzl", "rules_rust_dependencies", "rust_register_toolchains")
+
+rules_rust_dependencies()
+
+rust_register_toolchains(
+    allocator_library = "@test_cc_common_link//:allocator_library",
+    edition = "2021",
+    global_allocator_library = "@test_cc_common_link//:global_allocator_library",
+    versions = [
+        "nightly/2023-04-16",
```

Comment:
This should be in sync with `crates_repository`, and again, a version found in https://github.com/bazelbuild/rules_rust/blob/0.21.1/util/fetch_shas_NIGHTLY_ISO_DATES.txt should be used.